### PR TITLE
add github actions and issue templates

### PR DESCRIPTION
**Pull request by Emily-Jiang**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+name: MicroProfile Config Build
```
Comment (radcortez) [inline .github/workflows/build.yml:1]: Rename

**Review by radcortez — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    steps:
+      - uses: actions/checkout@v2.3.4
+        name: checkout
+        with:
+          ref: ${{ github.event.pull_request.merge_commit_sha }}
+
+      - uses: actions/setup-java@v2
+        name: set up jdk ${{matrix.java}}
+        with:
+          distribution: adopt
```
Comment (brunobat) [inline .github/workflows/build.yml:46]: Should we move to the new one?
```suggestion
          distribution: temurin
          java-version: 8
```

Comment (Emily-Jiang) [reply .github/workflows/build.yml]: The java-version was taken from the matrix [8, 11]

**Comment by Emily-Jiang** [comment]
@radcortez @brunobat please take another look to see whether you have any further comments

**Review by radcortez — APPROVED** [review]

**Pull request merged**